styles.css

```diff
@@ -30,7 +30,7 @@ button:hover {
 }
 
 button:active {
-  background: #008552;
+  background: #9acd32;
```

Comment:
- [ ] the menu works as expected but the close button is bigger than the one in the Figma design so please try to stick to the design and make it the same size Figma design one 
- Yours 
![Screen Shot 2022-10-18 at 12 30 51](https://user-images.githubusercontent.com/61790587/196418673-314c1220-9e07-4515-89dc-ce3f887da7a6.png)

- Figma
![Screen Shot 2022-10-18 at 12 33 54](https://user-images.githubusercontent.com/61790587/196418610-b969d005-a913-4d33-b50d-0b079b0475b1.png)